src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/AddThirdPartyIdpDialog.razor.cs

```diff
@@ -45,11 +45,11 @@ protected override void OnParametersSet()
 
     public async Task AddThirdPartyIdpAsync(EditContext context)
     {
-        if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
-        {
-            OpenErrorMessage(T("Please upload ThirdPartyIdp icon"));
-            return;
-        }
+        //if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
```

Comment:
Why commit it?

Reply:
Mr. Xie said the third-party platform icon upload is not required